docs/adding_a_new_microservice.md

```diff
@@ -0,0 +1,84 @@
+## Adding a new Microservice to Metaphysics
```

Comment:
I love this doc!!

One qualm I have is that Decoherence feels maybe a bit contrived...perhaps Impulse or some totally fictitious (or not so fictitious like 'auction results for an artist') might feel more natural?

Reply:
Yeah, I'll add a route like that, I just wanted to have something which we _could_ have, but don't so that you're not confused about the real routings for something ( otherwise I can just copypasta Convection's code into there )

Reply:
yea, I guess it would just be copying and pasting convection or impulse. I guess I'm biased cause I know what Decoherence is!

Maybe the auction results example is better suited? That seems easier to understand, and also something that we might have at one point (if Diffusion starts to offer that vs just an export).

Reply:
Wait, is decoherence a thing? There's no repo for it

Reply:
In the words of the great @dylanfareed 

![screen shot 2017-09-05 at 16 44 50](https://user-images.githubusercontent.com/49038/30082652-973d6190-9259-11e7-8c64-20c583184525.png)

I've swapped out the example name 🍡

Reply:
👋  😢

Reply:
lol, what a great thread :100: